osu.Game/Skinning/Editor/SkinEditor.cs

```diff
@@ -186,6 +197,13 @@ protected override void LoadComplete()
             SelectedComponents.BindCollectionChanged((_, __) => Scheduler.AddOnce(populateSettings), true);
         }
 
+        protected override void Dispose(bool isDisposing)
```

Comment:
nit: bit of a weird placement for a `Dispose()` method (would expect it to be dead last)